README.md

```diff
@@ -259,3 +269,12 @@ Weiere Infos zu den Objekt-Parametern von YForm befinden sich in der YForm-Doku.
 ### Einzelne Datensätze nicht in der sitemap.xml aufnehmen 
 
 Dazu kann man ein zusätzliches YFormfeld anlegen der das Indexieren speichert. Im Url-AddOn werden dann zwei Profile angelegt und auf das Index-Feld gefiltert. Das eine Profil erhält zusätzlich die Info "In Sitemap aufnehmen".
+
+## Debugging
+
+Ein Blick auf die Pathlist-Dateien `redaxo/cache/addons/yrewrite/pathlist.json` und  `/redaxo/cache/addons/url/pathlist.php` lässt Rückschlüsse darüber zu, ob die URLs korrekt generiert wurden oder ein Konfigurationsproblem vorliegt. Weitere Ansätze zum Debugging:
+
+* Sind alle gewünschten Domains in YRewrite vollständig und korrekt angegeben, einschl. separater Fehler 404-Seite?
+* Wurden Änderungen an der Datenbank vorgenommen, die sich mit dem Löschen des REDAXO-Caches, insb. der Pathlist-Dateien beheben lassen? Insb. Daten, die außerhalb von REDAXO geändert werden, benötigen ein Auffrischen des Caches.
```

Comment:
Die Urls liegen nicht mehr als Cache-Datei sondern direkt in der Datenbank vor. Der passt der Satz nicht ganz.